build_and_test.sh

```diff
@@ -0,0 +1,5 @@
+mvn package
```

Comment:
```suggestion
mvn clean package
```

Reply:
This fixes most of the Maven warnings about overlapping classes, but there is still one remaining that is caused by the added `graphviz-java-all-j2v8` dependency.